### DISCUSSION TITLE
Graphql Subscription with Apollo Client in Next.js build version

### DISCUSSION THREAD

**Discussion by alevelazcode** [vercel/next.js#46132] [Help]
this is my code of my Apollo client config
<img width="773" alt="image" src="https://user-images.githubusercontent.com/43557525/219975481-7abfc7a2-ea44-41c3-bb34-0548220acbba.png">

the subscription launches correctly and works perfect in development mode with 'npx dev' but in the version launched with 'npx build' and 'npx start' , the subscriptions and the websocket connection is never established. I have tried with the standalone build as the normal default build neither of the two generated versions of the project works the GraphQL subscriptions.

GraphQL's web socket and subscription connection to the server should work as well as in the development version with 'npx run dev' but it does not work in the production version generated with the build.
versions of my packages.
![image](https://user-images.githubusercontent.com/43557525/219984590-24aa1b37-cadb-4fb0-b05a-70b68f7d0858.png)
![image](https://user-images.githubusercontent.com/43557525/219984611-67b083e7-30b5-4647-9df3-39e64868b010.png)
![image](https://user-images.githubusercontent.com/43557525/219984629-e21be4f3-14ba-4929-a61e-054f7c4dcda8.png)

**Comment by mohsinht** [comment]
Did you find a solution to this ? I'm having a similar problem

**Comment by Maozerhouni** [comment]
Did you find a solution ?

**Comment by geoffraymond** [comment]
Anyone have a solution for this?
